### PR TITLE
Adding export script in package.json

### PR DESCRIPTION
**Pull request by s2sharpit**
## Related Issue

- Information about the related issue

Adding scripts :
``` 
    "export": "next build && next export",
    "prepare": "husky install",
```
Closes: #308 

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by vinzvinci — COMMENTED** [review]
This error is showing
image

**Comment by Panquesito7** [comment]
`husky` might need to be installed if it's not, I guess.

**Comment by vinzvinci** [comment]
> `husky` might need to be installed if it's not, I guess.

it was added already on `prepare`, maybe need to remove temporarily and `npm i` again then add the `"prepare": "husky install"` again then run `npm i` again

**Comment by vinzvinci** [comment]
package is missing 🤣

**Review by vinzvinci — APPROVED** [review]
LGTM 🎉

**Comment by s2sharpit** [comment]
Hey, 
I have added the package husky in package.json and I think it's working fine now.

File: next.config.mjs [hunk 1]
```diff
@@ -1,12 +1,10 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-  experimental: {
-    appDir: true,
```
Comment (s2sharpit) [inline next.config.mjs:5]: Not needed to add appDir in experimental after Next.js 13.4.0 version.

**Review by vinzvinci — APPROVED** [review]
LGTM! 

Merging this now on behalf of reviewers!

**Pull request merged**